templates/aggkit/aggkit-config.toml

```diff
@@ -94,6 +94,8 @@ polygonZkEVMAddress = "{{.sovereign_rollup_addr}}"
 polygonZkEVMAddress = "{{.zkevm_rollup_address}}"
 {{- end }}
 
+BridgeAddr = "{{.zkevm_bridge_address}}"
```

Comment:
`polygonBridgeAddr = "{{.zkevm_bridge_address}}"` - this needs to be removed before merging